settings.gradle

```diff
@@ -1 +1,19 @@
+plugins {
+    id "com.gradle.enterprise" version "3.14.1"
+}
+
+def runsOnCI = System.getenv("CI") == 'true'
+
+gradleEnterprise {
+    buildScan {
+        termsOfServiceUrl = "https://gradle.com/terms-of-service"
+        termsOfServiceAgree = "yes"
+
+        publishAlwaysIf(runsOnCI)
+    }
+}
+
 rootProject.name = "candlepin"
+
+include 'client'
+include 'spec-tests'
```

Comment:
Are these needed? I don't think the 4.1 branch has these projects

Reply:
no, i was just copying it from main and not noticed that. Done